extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -231,6 +239,17 @@ export class AmpA4A extends AMP.BaseElement {
     this.experimentalNonAmpCreativeRenderMethod_ =
       platformFor(this.win).isIos() ? XORIGIN_MODE.SAFEFRAME : null;
 
+    /**
+     * Gets a notion of current time, in ms.  The value is not necessarily
+     * absolute, so should be used only for computing deltas.  When available,
+     * the performance system will be used; otherwise Date.now() will be
+     * returned.
+     *
+     * @const {function():number}
+     * @private
+     */
+    this.getNow_ = (this.win.performance && this.win.performance.now) ?
```

Comment:
I'm curious if you could have saved yourself the bind had you done:

`this.getNow_ = () => {
    return (this.win.performance && this.win.performance.now) ? this.win.performance.now : Date.now;
}`